src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,56 @@
+package seedu.addressbook.data.tag;
+
+import java.util.ArrayList;
+import seedu.addressbook.data.peron.Person
+
+/**
+ * To display during exiting of the program
+ * the addition and removal of tags during the particular session
+ */
+public class Tagging {
+
+    public static final ArrayList<String> ALL_TAG_CHANGES = new ArrayList<String>();
+   
+    private final static String MESSAGE = "Tag Changes made during the session:";
+    private final static String EXAMPLE = "+ Jake Woo [friend]";
+    private final static String PREFIX_ADD = "+";
+    private final static String PREFIX_REMOVE = "-";
+    private final static String LS = System.lineSeparator();
+    
+    /**
+     * Creates a Tagging Object
+     */
+    public Tagging(Person person, Tag tag, boolean isAdded){
+    	final String prefix = isAdded ? PREFIX_ADD : PREFIX_REMOVE;
```

Comment:
While this is a good attempt, I can't agree that this is a good Association Class. Why? because even though you are accepting the Person and the Tag in the constructor you are not maintaining it within the class. This way later on when we want additional information about the two associated classes we will not have it.

Ways to improve.
1. add private Person and Tag instance variables to Tagging.java
2.Instantiate them in the constructor 
3.set getters for those private variables